src/umpire/alloc/MallocAllocator.hpp

```diff
@@ -69,11 +69,21 @@ struct MallocAllocator {
 #if defined(UMPIRE_ENABLE_CUDA)
     int pageableMem = 0;
     int cdev = 0;
-    cudaGetDevice(&cdev);
+    cudaError_t err = cudaGetDevice(&cdev);
+    if (err != cudaSuccess) {
+      UMPIRE_ERROR(umpire::runtime_error,
+                   fmt::format("Error when trying to get CUDA Device: {}", cudaGetErrorString(err)));
+    }
 
     // Device supports coherently accessing pageable memory
     // without calling cudaHostRegister on it
-    cudaDeviceGetAttribute(&pageableMem, cudaDevAttrPageableMemoryAccess, cdev);
+    err = cudaDeviceGetAttribute(&pageableMem, cudaDevAttrPageableMemoryAccess, cdev);
+    if (err != cudaSuccess) {
+      UMPIRE_ERROR(runtime_error,
+                   fmt::format("cudaDeviceGetAttribute( pageableMem = {}, cudaDevAttrPageableMemoryAccess = {}, cdev = "
```

Comment:
Could you use consistent spacing next to open and closing parentheses in the error message? At the beginning there is an extra space and at the end there is not.